daemon/storage.go

```diff
@@ -446,6 +446,12 @@ func (s *BtrfsStorage) PrepareContainer(containerId, sharedDir string, readonly
 	if err := syscall.Mount(btrfsRootfs, mountPoint, "bind", syscall.MS_BIND, ""); err != nil {
 		return nil, fmt.Errorf("failed to mount %s to %s: %v", btrfsRootfs, mountPoint, err)
 	}
+	if readonly {
```

Comment:
so there may be a security problem among these two mount operations. The early existing container can possible write to the roofs of this one.